internal/gateway/statistics/statistics.go

```diff
@@ -0,0 +1,69 @@
+package statistics
+
+import (
+	"fmt"
+	"os/exec"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+type Counters struct {
+	ChunksAdded          int64  `json:"Publish.n_chunks_added,string"`
+	ChunksDuplicated     int64  `json:"Publish.n_chunks_duplicated,string"`
+	CatalogsAdded        int64  `json:"Publish.n_catalogs_added,string"`
+	UploadedBytes        int64  `json:"Publish.sz_uploaded_bytes,string"`
+	UploadedCatalogBytes int64  `json:"Publish.sz_uploaded_catalog_bytes,string"`
```

Comment:
I am unsure about the use of `Publish.something`. The use of the `.` inside JSON keys is not very well supported in general (in js you can access a field by using the `.`)

Wouldn't it be simpler to have a structure like.

```json
{"lease/path": {
     "publish": {
        "n_chunks_added": 5,
        ...
    }, 
     "start_time": "some start time"}}
```
Or maybe even delete the `publish` namespace?

Reply:
Yeah, we can definitely do it without the "Publish." prefix. We just need to change it in the receiver as well.